src/basic/cgroup-util.c

```diff
@@ -2468,6 +2468,14 @@ static int cg_unified_update(void) {
 
         struct statfs fs;
 
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+                /* msan somehow thinks fs in unitialized, even though statfs(2)
+                 * is clear that all fields are set:
+                 * https://github.com/systemd/systemd/issues/8474
+                 */
+                zero(fs);
```

Comment:
I've just realized that `fuzz-unit-file` tries to trim `/sys/fs/cgroup` by deleting everything there, which it probably shouldn't do especially when it is run as `root`. I think that instead of getting around the issue reported by `msan` it would be better to fix `fuzz-unit-file` so that it won't try to remove anything that hasn't been created by it.